apps-rendering/src/components/Standfirst/Standfirst.defaults.tsx

```diff
@@ -0,0 +1,104 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import {
+	background,
+	border,
+	text,
+} from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign, ArticleDisplay } from '@guardian/libs';
+import { remSpace } from '@guardian/source-foundations';
+import { map, withDefault } from '@guardian/types';
+import { standfirstBackgroundColour } from 'editorialStyles';
+import type { Item } from 'item';
+import { getFormat } from 'item';
+import { pipe } from 'lib';
+import type { FC, ReactElement, ReactNode } from 'react';
+import { renderStandfirstText } from 'renderer';
```

Comment:
I think we might want to consider extracting this rendering code from `renderer.ts` and placing it somewhere in the standfirst component folder? Possibly as a `StandfirstText.tsx` component or something along those lines? Again, not a blocker, perhaps a follow-up?